GRIB2_Template_4_148_ProductDefinitionTemplate_en.csv

```diff
@@ -0,0 +1,42 @@
+"Verification scores for individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer at a point in time",49,1,Type of additional arguments for verification score,(see Code table 4.122),,4.122,,Operational
+"Verification scores for individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer at a point in time",50,1,Number of additional arguments for verification (NA),,,,,Operational
+"Verification scores for individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer at a point in time", , ,The next two entries are repeated NA times na=0:NA – can be 0 times if no parameters are necessary (na=0),,,,,Operational
+"Verification scores for individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer at a point in time",51 + (na-1)*5,1,Scale factor of additional argument for verification,,,,,Operational
+"Verification scores for individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer at a point in time",(52 + (na-1)*5) - (55 + (na-1)*5),4,Scaled value of additional argument for verification,,,,,Operational
+"Verification scores for individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer at a point in time",(51 + NA*5) - (52 + NA*5),2,Year of start of verification period,,,,,Operational
+"Verification scores for individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer at a point in time",53 + NA*5,1,Month of start of verification period,,,,,Operational
+"Verification scores for individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer at a point in time",54 + NA*5,1,Day of start of verification period,,,,,Operational
+"Verification scores for individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer at a point in time",55 + NA*5,1,Hour of start of verification period,,,,,Operational
+"Verification scores for individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer at a point in time",56 + NA*5,1,Minute of start of verification period,,,,,Operational
+"Verification scores for individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer at a point in time",57 + NA*5,1,Second of start of verification period,,,,,Operational
+"Verification scores for individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer at a point in time",58 + NA*5,1,Number of verification period time ranges (NV),,,,,Operational
+"Verification scores for individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer at a point in time", , ,The next three entries are repeated NR times nv=1:NV,,,,,Operational
```

Comment:
* NR is not defined
* confirm that "next three entries" is correct
* add End of sequence after?